scripts/fetch/fetch.view_limits.R

```diff
@@ -0,0 +1,13 @@
+fetch.view_limits <- function(viz = as.viz('view-limits')){
+  bbox <- viz[['bbox']]
+  sp.bbox <- as.sp_box(xs = bbox[c(1,3)], ys = bbox[c(2,4)], proj.string = sp::CRS("+proj=longlat +datum=WGS84"))
+  bbox.transformed <- sp::spTransform(sp.bbox, viz[['proj.string']])
+  
+  view.limits <- get_sp_lims(bbox.transformed, # it would be nice to just use defaults if these aren't included:
+                             width = viz[['width']], height = viz[['height']], pointsize = viz[['pointsize']])
+  
+  out <- append(view.limits, list(proj.string = viz[['proj.string']],
+    width = viz[['width']], height = viz[['height']], pointsize = viz[['pointsize']]))
```

Comment:
Note that this is more than view limits. Maybe consider renaming to view_properties?

Reply:
yeah...I am not really sure how to do this part of it. I want to set up the view early, since so much data getting relies on it, but it is kind of overloaded.

Reply:
Run with it! I like putting this all in one place.